aggsender/flows/flow_aggchain_prover.go

```diff
@@ -87,7 +88,15 @@ func (a *AggchainProverFlow) CheckInitialStatus(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("aggchainProverFlow - error getting last sent certificate: %w", err)
 	}
-	return a.sanityCheckNoBlockGaps(lastSentCertificate)
+	err = a.sanityCheckNoBlockGaps(lastSentCertificate)
+	if a.requireNoFEPBlockGap {
```

Comment:
This is more cleaner to me, but its just a suggestion. I would change the log a bit though:

```suggestion
	if err := a.sanityCheckNoBlockGaps(lastSentCertificate); err != nil {
	     if a.requireNoFEPBlockGAP {
	           return err
	     }
	     
	     a.log.Warnf("aggchainProverFlow - ignoring block gaps due to RequireNoFEPBlockGap config. Err: %w", err}
```